src/lib_ccx/params.c

```diff
@@ -1114,6 +1113,15 @@ int parse_parameters (struct ccx_s_options *opt, int argc, char *argv[])
 			version(argv[0]);
 			return EXIT_WITH_HELP;
 		}
+
+		if(!strcmp(argv[i], "-man"))	//Generate manual page
+		{
+#ifdef __linux__
+			system("bash manscript.sh");
+			mprint("Finished executing man script\n");
+#endif
+			return EXIT_WITH_HELP;
+		}
 		if (strcmp (argv[i], "-")==0 || strcmp(argv[i], "-stdin") == 0)
 		{
 #ifdef WIN32
```

Comment:
Minimum two spaces were required after every parameter so that help2man is able to detect different options.